src/Raven.Studio/typescript/models/database/tasks/ongoingTaskOlapEtlTransformationModel.ts

```diff
@@ -74,6 +76,31 @@ class ongoingTaskOlapEtlTransformationModel {
             aceValidation: true
         });
 
+        const checkScriptName = (val: string,
+                                 params: any,
+                                 callback: (currentValue: string, errorMessageOrValidationResult: string | boolean) => void) => {
+            if (this.name()) {
+                new validateNameCommand('Script', val)
+                    .execute()
+                    .done((result) => {
+                        if (result.IsValid) {
+                            callback(this.name(), true);
+                        } else {
+                            callback(this.name(), result.ErrorMessage);
+                        }
+                    })
+            }
```

Comment:
do we want to call `callback` then name is null or empty?

Reply:
there is a check:  ` if (this.name())`  few lines above

Reply:
yes I know but the question is if we should call callback even when name is empty?

Reply:
added `required: true` so now this callback is not being called if empty